readme.md

```diff
@@ -116,7 +147,7 @@ How is rebase different than a merge?
 
 Your Answer:
 ```text
-
+Both git Merge and git Rebase integrate changes from one branch into another, but they do it in different ways. Merging is referred to as the "non-descrtuvctive" option because when you merge two branches (say featureBranch with master) you can see the the commit history of both branches by running 'git log'. Rebasing is different, because when you would merge these two branches it pushes the feature branch on top of the master, re-writing the project history by making new commits for each commit in the master branch.
```

Comment:
your description is correct but remove the word "merge" from the rebase description